README.md

```diff
@@ -12,7 +12,7 @@ To use this package, please [install the Pulumi CLI first](https://pulumi.io/).
 
 This package is available in many languages in the standard packaging formats.
 
-### Node.js (Java/TypeScript)
+### Node.js (Java-/TypeScript)
```

Comment:
```suggestion
### Node.js (JavaScript/TypeScript)
```

I guess we don't have to save keystrokes here :)

Reply:
👍